backend/app/src/main/kotlin/io/tolgee/controllers/PublicController.kt

```diff
@@ -94,14 +87,25 @@ class PublicController(
     userAccountService.setResetPasswordCode(userAccount, code)
     val message = SimpleMailMessage()
     message.setTo(request.email!!)
-    message.subject = "Password reset"
+
     val callbackString = code + "," + request.email
     val url = request.callbackUrl + "/" + Base64.getEncoder().encodeToString(callbackString.toByteArray())
-    message.text = """Hello!
- To reset your password click this link: 
+    if (userAccount.accountType == UserAccount.AccountType.THIRD_PARTY) {
+      message.subject = "Initial password configuration"
+      message.text = """Hello!
+To set a password for your account, please go to the following link: 
+$url
+
+If you have not requested this e-mail, please ignore it."""
+    } else {
+      message.subject = "Password reset"
+      message.text = """Hello!
+To reset your password click this link: 
 $url
 
- If you have not requested password reset, please just ignore this e-mail."""
+If you have not requested password reset, please just ignore this e-mail."""
+    }
+
     message.from = properties.smtp.from
```

Comment:
Please do it like here: io.tolgee.component.email.EmailVerificationSender.sendEmailVerification